gradle/bintray/bintray.gradle

```diff
@@ -31,17 +31,20 @@ artifacts {
 
 // Bintray
 Properties properties = new Properties()
-// use bintray properties either locally or from system environments
+def bintrayUser
+def bintrayApiKey
 if (project.rootProject.file('local.properties').exists()) {
     properties.load(project.rootProject.file('local.properties').newDataInputStream())
+    bintrayUser = properties.getProperty("bintray.user")
+    bintrayApiKey = properties.getProperty("bintray.apikey")
 } else {
-    properties.setProperty('bintray.user', System.getenv('BINTRAY_USER') ?: '')
-    properties.setProperty('bintray.apikey', System.getenv('BINTRAY_APIKEY') ?: '')
+    bintrayUser = bintrayUser = System.getenv("BINTRAYUSERNAME")
```

Comment:
do you have any reason why you've done with uppercasing instead of camelCase like in etensions?

Reply:
Convention for secrets in Github is to set the name in uppercase and while it's probably case-insensitive because it should work on extensions, I wanted to be sure 😄

Reply:
well then i would add underscores so human can read that :D

Reply:
Changed in 91d9cd2a778ab7272cd38a476f81691a8850f976